docs/mobile-apps/automated-testing/appium/appium-versions.md

```diff
@@ -172,6 +171,40 @@ The following list of custom Appium plugins are supported:
     </tr>
   </thead>
   <tbody>
+     <tr>
+      <td>
+        <code>stable</code>
+      </td>
+      <td>
+        <span className="sauceGreen">Active</span>
+      </td>
+      <td>
+        This is a our LTS image with drivers that we only change when major functionality breaks or after 31.08.2026.<br />
```

Comment:
lets resolve the abbreviation

Reply:
lets add here a 12 months EOL date

Reply:
That would be equal to our regular images, but sure I will do that